integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala

```diff
@@ -202,25 +210,37 @@ object BenchUtils {
         val end = System.nanoTime()
         val elapsed = NANOSECONDS.toMillis(end - start)
         queryTimes.append(elapsed)
-        println(s"$logPrefix Iteration $i took $elapsed msec.")
+
+        val status = if (taskFailureListener.taskFailures.isEmpty) {
+          STATUS_COMPLETED
+        } else {
+          // add the first task failure to the list of exceptions in the report
+          exceptions.append(taskFailureListener.taskFailures.head.toString)
```

Comment:
do we need to/want to change the suffix on the file at all?   I think adding this will make the file name show up with -failed right?

Reply:
ie line 239 below